### ISSUE TITLE
Please add support for EPPlus.Core

### ISSUE THREAD

**Issue by XdX-Software** [ipvalverde/EPPlus.DataExtractor#2]
Thank you and kind regards

**Comment by ipvalverde** [comment]
Good point.
I'll add support to EPPlus.Core

**Comment by trbngr** [comment]
Any update on this?

**Comment by ipvalverde** [comment]
I noticed that the official EPPlus package is now targeting .netstandard, I thought it would be better to just wait for a stable version of the EPPlus 4.5.

I can try to come out with a beta version as well targeting their prerelease version of 4.5.X.

**Comment by trbngr** [comment]
Just sent a PR.

**Comment by ipvalverde** [comment]
Thank you @trbngr for the PR!

I just adapted the nuspec for the nuget package generation. I'm still having problems with AppVeyor, I didn't have enough time to actually learn how it works, I have to change the build configuration files.

I've manually uploaded the new version as a pre-release: https://www.nuget.org/packages/EPPlus.DataExtractor/2.0.0-alpha0001


I'll try to find some time to adapt it to also target the older versions of .net and EPPlus.
